docs/source/create_channel/create_channel.md

```diff
@@ -11,11 +11,24 @@ In the process of creating the channel, this tutorial will take you through the
 - [Joining peers to the channel](#join-peers-to-the-channel)
 - [Setting anchor peers](#set-anchor-peers)
 
-## Setting up the configtxgen tool
 
-Channels are created by building a channel creation transaction and submitting the transaction to the ordering service. The channel creation transaction specifies the initial configuration of the channel and is used by the ordering service to write the channel genesis block. While it is possible to build the channel creation transaction file manually, it is easier to use the [configtxgen](../commands/configtxgen.html) tool. The tool works by reading a `configtx.yaml` file that defines the configuration of your channel, and then writing the relevant information into the channel creation transaction. Before we discuss the `configtx.yaml` file in the next section, we can get started by downloading and setting up the `configtxgen` tool.
+## Before you begin
+
+**Important:** This tutorial uses the Fabric test network and is compatible with v2.2 or lower of the
+test network sample. After you have installed the [prerequisites](../getting_started.html),
+**you must run the following command** to clone the required version of the
+[hyperledger/fabric samples](https://github.com/hyperledger/fabric-samples) repository and checkout
+the correct version tag. The command also installs the Hyperledger Fabric platform-specific
+binaries and config files for the version into the `/bin` and `/config` directories of `fabric-samples`
+so that you can run the test network.
+
+```
+curl -sSL https://bit.ly/2ysbOFE | bash -s -- 2.2.1 1.4.9
```

Comment:
```suggestion
curl -sSL https://bit.ly/2ysbOFE | bash -s -- 2.2.2 1.4.9
```

v2.2.2 has been releases and tagged in fabric-samples as well.

Reply:
done